lms/templates/email/instructor_email_digest/body.html.jinja2

```diff
@@ -0,0 +1,759 @@
+<!doctype html>
```

Comment:
I've just copy-pasted this HTML & CSS from the Mailchimp template that we were using. I haven't modified it other than to fix the formatting/indentation and to change it from Mailchimp's template language to Jinja2.